cw_bitcoin/lib/electrum_wallet_addresses.dart

```diff
@@ -63,7 +63,11 @@ abstract class ElectrumWalletAddressesBase extends WalletAddresses with Store {
   }
 
   @override
-  set address(String addr) => null;
+  set address(String addr) {
+    final addressRecord = addresses.firstWhere((addressRecord) => addressRecord.address == addr);
+    receiveAddresses.remove(addressRecord);
+    receiveAddresses.insert(0, addressRecord);
```

Comment:
can you explain what this function does?

Reply:
It updates the primary address for receiving funds in the wallet to the one selected by the user.